platform/src/lib.rs

```diff
@@ -44,15 +44,21 @@ extern "C" {
     #[link_name = "roc__mainForHost_0_size"]
     fn size_Fx() -> i64;
 
-    #[link_name = "roc__mainForHost_0_result_size"]
-    fn size_Fx_result() -> i64;
+    // #[link_name = "roc__mainForHost_0_result_size"]
+    // fn size_Fx_result() -> i64;
```

Comment:
Assuming this needs to stay commented out, can you add another comment with the reason for that @lukewilliamboswell?

Reply:
I removed it because it isn't used.